x.consolidation/consolidation.md

```diff
@@ -0,0 +1,517 @@
+uses almost double as much block space as him.  The feerate data in the
+appendix to this chapter indicates such savings are possible under our
+rough assumptions over 96% of the time.
+
+If Bob needed to use more than two inputs per transaction in order to create
+a changeless transaction, Alice would be able to save more money even if she
+used a correspondingly-higher feerate for her consolidation transaction.
+
+That said, a clear advantage of changeless transactions over
+consolidation transactions is that they allow the spender to save block
+space and achieve modest fee savings while preserving their privacy
+against the problems described in the *Consolidation Privacy* section
+above.  For most individual spenders or even small businesses who only
+spend infrequently, that privacy advantage is more important to them than
+the fee savings from consolidation.
+
+## Predictive/Organic UTXO management
+
+[Predictive UTXO management][] is a service provided by BitGo that
+"reduces the overall cost by minimizing transaction sizes at high fee
+rates, while automatically sweeping up and processing many small
+fragments of coins when fees are low."  The exact implementation of 
+this technique is proprietary, but it's probably similar to [organic
+UTXO consolidation][] as described by Dmitry Petukhov.
+
+This class of techniques eliminates the extra block chain space used by
+consolidation transactions, allowing it to possibly be more efficient
+and save more on fees.  It may also be easier to implement for some
+services, as it doesn't require waiting hours or days for consolidation
+transactions to confirm.  Additionally, BitGo's predictive UTXO
+management system in particular seems to achieve some of its savings
+using the changeless transactions described in the previous section,
+with the extra savings and privacy benefits described there.
+
+However, when it comes to saving fees, UTXO consolidation will
+frequently save more than predictive/organic techniques for spenders who
+want their spending transactions to confirm within the next several blocks.
+That's because saving a few dozen bytes in block chain space using
+predictive/organic transactions doesn't match being able to save a large
+percentage of UTXO spending costs by spending UTXOs at a lower feerate.
+This was actually quantified in a earlier illustration: 
+
+![Savings by consolidation feerate](img/estimating-savings-by-feerate.png)
+
+For those that spend an average of two UTXOs or more per transaction
+and who can consolidate at 50% or less of their normal feerate (and who
+use the P2WPKH transactions analyzed here), consolidation at least
+breaks even on costs and can easily save more money than sending a
+normal transaction.  The feerate data included in the appendix to this
+chapter shows that saving at least 50% on feerates for a nominal 2-block
+confirmation target was possible 99.85% of the time during our analysis
+period of November and December 2019.
```

Comment:
How difficult is it to expand this analysis to years instead of months?

Reply:
Trivially easy, but I can only collect data going forward, so I won't have a full year of data until mid-October 2020.

Reply:
Ill resolve this comment for now and we can revisit in the future when it makes sense.